bedrock/firefox/tests/test_views.py

```diff
@@ -198,19 +198,19 @@ def test_send_android_sms(self):
         })
         ok_(resp_data['success'])
         self.mock_send_sms.assert_called_with('post', 'subscribe_sms', data={
-            'mobile_number': '15558675309',
+            'mobile_number': '5558675309',
             'msg_name': views.SEND_TO_DEVICE_MESSAGE_SETS['default']['sms']['android'],
             'lang': 'en-US',
         })
 
     def test_send_android_sms_non_en_us(self):
         resp_data = self._request({
             'platform': 'android',
-            'phone-or-email': '5558675309',
+            'phone-or-email': '015558675309',
```

Comment:
Not necessarily. The basket side will format it appropriately for the country, but we're doing geolocation so that we can interpret local numbers as well. I mainly needed this so that I could have a test that made sure that a longer phone number would pass validation.